src/scripts/main.js

```diff
@@ -354,7 +354,49 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.forEach((person) => {
+  const row = document.createElement('tr');
 
-// write your code here
+  const nameCell = document.createElement('td');
+
+  nameCell.textContent = person.name;
+
+  const genderCell = document.createElement('td');
+
+  genderCell.textContent = () => {
```

Comment:
Just a ternary expression will be enough, no need to complicate things here :) 

![m](https://github.com/mate-academy/js_task_generate_table_DOM/assets/57567908/0ab44d75-d69d-44d8-9675-81caab6d0237)